src/main/java/frc/robot/RobotContainer.java

```diff
@@ -212,66 +216,34 @@ private void setupDriver() {
 
         driver.rightTrigger().and(climb.reachedClimberStart)
             .onTrue(climb.runClimberMotorCommand(climb.passedClimbAngle()));
+
+        // remove later
+        SmartDashboard.putNumber("elevatorTargetHeight", 20);
+        driver.a().whileTrue(elevator
+            .moveToMagic(() -> Inches.of(SmartDashboard.getNumber("elevatorTargetHeight", 20))));
```

Comment:
Lets take this off driver, move to pit operator or its own controller. I want to keep driver/operator clean of testing controls.